### DISCUSSION TITLE
ScriptReference/GameObject-activeInHierarchy

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#202] [Notes]
# ScriptReference/GameObject-activeInHierarchy

https://docs.unity3d.com/ScriptReference/GameObject-activeInHierarchy.html

**Comment by mvi** [comment]
Note that this will return false if used on an uninstantiated prefab (for example if you're writing an editor tool that requires selecting a prefab in the project window).  
For example, this will always be false when a prefab is selected in the Project window:  
  
`Selection.activeGameObject.activeInHierarchy`
